recipe/meta.yaml

```diff
@@ -11,17 +11,17 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - antlr ==4.8
   host:
-    - python >=3.6,<3.9
+    - python >=3.6,<3.10
     - pip
   run:
-    - python >=3.6,<3.9
+    - python >=3.6,<3.10
     - omegaconf >=2.0.2
```

Comment:
can you update the omegaconf dependency to:
```
omegaconf>=2.0.5,<2.1
```
Per the [requirements](https://github.com/facebookresearch/hydra/blob/1.0_branch/requirements/requirements.txt) of Hydra 1.0?

Also, importlib_resources is only needed for Python < 3.9.